composer.json

```diff
@@ -12,7 +12,7 @@
         "doctrine/dbal": "^2.11 || ^3",
         "doctrine/persistence": "^1.3.0 || ^2",
         "blueimp/jquery-file-upload": "9.*",
-        "mapbender/data-source": "~0.1.24 || ^0.2",
+        "mapbender/data-source": "dev-feature/enableDBSIMMSupport",
```

Comment:
please revert this change, relying on a feature branch of another repo from the main branch is not a good idea

Reply:
Of course, this has to be adapted in case of merging. I just need the feature branches like this for the moment, since I rely on these changes and that unfortunately includes the dependency. What would be the appropriate way of handling this?

Reply:
probably releasing new versions of all packages at the same time. A `composer update` will then update all of the dependencies